modules/data.atmosphere/R/ERA5_download.R

```diff
@@ -0,0 +1,142 @@
+#' @description
+#' This function helps to download the yearly ERA5 data based on the prescribed features using the CDS API.
+#' @title ERA5_cds_annual_download
+#' 
+#' @param years Numeric: a series of years to be downloaded (e.g., 2012:2021).
+#' @param months List: a list contains months to be downloaded (e.g., list("01", "02") to download files in Jan and Feb).
+#' @param days List: a list contains days to be downloaded (e.g., list("01", "02") to download files in the first and second days).
+#' @param times List: a list contains times to be downloaded (e.g., list('00:00','03:00') to download files for the times 12:00 and 3:00 am UTC).
+#' @param area Character: a string contains the bounding box (formatted as "North/West/South/East") to be downloaded (e.g., "85/-179/7/-20").
```

Comment:
This is a really odd format. Even if this is what the internal API expects, I think it would be a better design to take the bounding box as a vector of numbers and then format this string inside your function.

Reply:
Fixed.